e2e/e2etests/test_precompiles_bank.go

```diff
@@ -59,7 +60,7 @@ func TestPrecompilesBank(r *runner.E2ERunner, args []string) {
 	require.Equal(r, uint64(0), cosmosBalance.Uint64(), "spender cosmos coin balance should be 0")
 
 	// Approve allowance of 500 ERC20ZRC20 tokens for the bank contract. Should pass.
-	tx, err := r.ERC20ZRC20.Approve(r.ZEVMAuth, bank.ContractAddress, depositAmount)
+	tx, err := r.ERC20ZRC20.Approve(r.ZEVMAuth, fungibletypes.ModuleAddressZEVM, depositAmount)
```

Comment:
I think the allowance should still be done to the interfaced percompile to more friendly DevEx

You interact with bank to deposit: you approve bank, bank more the tokens to fungible
And same for staking

Reply:
Not sure to understand, shouldn't approve bank precompile address here based on discussed changes?

Reply:
Implemented in 98fb634cef3e492758a3860515d8622b23601bdb, it was on the works.